extras/userdoc/generator/README.md

```diff
@@ -0,0 +1,24 @@
+# README for the new NEST Help Generator
```

Comment:
Remove "new". No one will ever change this if it becomes old one day. Make "Help Generator" all lower case